test/mail_helper.js

```diff
@@ -1,101 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-var MailParser = require('mailparser').MailParser
-var restify = require('restify')
-var config = require('rc')(
-  'fxa_mail_helper',
```

Comment:
Huh, was this leftover code coped from fxa-auth-server that isn't used for the tests in this repository?